### DISCUSSION TITLE
Errors continue to occur during GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by jihyunlee2023** [community/community#114563] [GitHub Education]
### Select Topic Area

Question

### Body

<img width="315" alt="스크린샷 2024-03-28 오후 1 25 54" src="https://github.com/community/community/assets/152181713/7df760fd-ba87-46a7-a43d-2764451cb1be">

I can't resolve this issue. Although I've received student approval, I'm unable to access the service. I haven't received any emails either.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @jihyunlee2023! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
